DuckDuckGo/RemoteMessaging/RemoteMessagingClient.swift

```diff
@@ -47,9 +47,9 @@ final class RemoteMessagingClient: RemoteMessagingProcessing {
         static let minimumConfigurationRefreshInterval: TimeInterval = 60 * 30
         static let endpoint: URL = {
 #if DEBUG
-            URL(string: "https://raw.githubusercontent.com/duckduckgo/remote-messaging-config/main/samples/ios/sample1.json")!
+            URL(string: "https://www.jsonblob.com/api/1316017217598578688")!
```

Comment:
⚠️ This shouldn’t be merged; after approval, I will revert this file to the original URLs.